build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -12,7 +12,7 @@ public class Crossgen
     {
         private string _coreClrVersion;
         private string _crossGenPath;
-        private static readonly string[] s_excludedLibraries = 
+        private static readonly string[] s_excludedLibraries =
         {
             "mscorlib.dll",
```

Comment:
Please include System.Private.CoreLib.dll and System.Private.CoreLib.ni.dll in the exclude list since they come from CoreCLR package as well.